### DISCUSSION TITLE
Update uBO README + Who I am/was

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2596] [General] [answered]
@gorhill

I am the same user that used to have the name `q1800` and helped with the uBO project by submitting patches to fix/change code, updating documentation such as the README.md files for the uBO repository, updating issue templates, and other various documentation. I also previously made tons of extensive edits to the uBO Wiki.

Over three months ago, my account got hijacked by someone I know personally, and I couldn't get it back. I had the account completely closed and deleted so no one could use it or do anything malicious.

So, long story short, I am/was `q1800`, and this is my new username. I want to continue helping out with uBO however I can.

---

The first commit I would like to submit is the following:

https://github.com/GH-User999/uBlock/commit/1816aba217da9999c5c4e1532460422e3b2425b3

Here is a summary of some of the changes and my reasoning:

I merged the initial brief description paragraph with the general information section because several sentences were complete duplicates or partially had the same or similar verbiage. Also, having two separate parts for the same purpose and stating the same message didn't make sense.

I also merged both "Do NOT" statements in the All Browsers section because they stated similar things.

---

Also, here is another idea that I would like to propose to you and get your thoughts on:

What do you think about removing the reference to uMatrix in the following paragraph and trimming the sentence?

`Ads, "unintrusive" or not, are just the visible portion of the privacy-invading means entering your browser when you visit most sites. uBO's primary goal is to help users neutralize these privacy-invading methods in a way that welcomes those users who do not wish to use more technical, involved means (such as uMatrix).`

The uMatrix project/repository was archived almost two years, and I believe you no longer mention/recommend it.

Thank you for everything!

**Comment by gorhill** [comment, marked as answer]
Perfect, thanks. I pulled everything before realizing I didn't answer re. uMatrix. You're right, best to ~~remove~~trim this sentence.

**Reply by ghost** [reply]
@gorhill

Here is the PR to remove the uMatrix reference, trim the sentence, and remove the uMatrix URL from the reference links.

https://github.com/gorhill/uBlock/pull/3887

Thank you

**Comment by ghost** [comment]
Maybe it's a good idea to require 2FA in organization?

**Reply by iam-py-test** [reply]
GitHub's already requiring it for most people, and I think they plan on making it required some time this year/next year

**Comment by Yuki2718** [comment]
Tho I appreciate all your hard work, I personally think these sentences should be kept:
> uBO is **NOT** an "ad blocker";
> Without the preset filter lists, this extension is nothing.

**Reply by iam-py-test** [reply]
>  Without the preset filter lists, this extension is nothing.

I disagree this (not that my opinion matters). Even without the default lists, users can still use dynamic filtering and add rules to "my filters". It isn't nothing, it just doesn't have anything to tell it what to do.

**Reply by Yuki2718** [reply]
Maybe there's better wording. But I feel large portion of misbehavior regarding blocker (e.g. using multple blocker, not reporting issues to filter author) mainly stem from the lack of understanding that blocker simply forces human-curated filter rules.

**Reply by gorhill** [reply]
> Without the preset filter lists, this extension is nothing

Dynamic filtering is still available without filter lists, that would be the equivalent of RequestPolicy/sort of NoScript, so it's not "nothing". The filter lists allows uBO to be an install-and-forget content blocker, so still a big contribution.

**Reply by Yuki2718** [reply]
For the first sentence, I have recently read and tweeted a blog post complaining that the common - yeah, really really common - practice of calling uBO as an adblocker is causing underestimate of what it can do, which I agree to. Couldn' find tweet but gorhill once said somewhere he has to keep correcting statements calling uBO as an adblocker.

**Reply by gorhill** [reply]
We still do not call it an "ad blocker", it says _wide-spectrum content blocker_. We can't prevent people out there from calling uBO an _"ad blocker"_, it's out of our control regardless of how we phrase it in the README.

**Reply by Yuki2718** [reply]
I mean explicitly denying what it is is stronger than only showing correct definition. And people can reference this as they claim uBO is not an adblocker.
